pkg/machines/constants/provider-action-types.js

```diff
@@ -49,6 +49,7 @@ export const GET_STORAGE_POOL = "GET_STORAGE_POOL";
 export const GET_STORAGE_POOLS = "GET_STORAGE_POOLS";
 export const GET_STORAGE_VOLUMES = "GET_STORAGE_VOLUMES";
 export const GET_VM = "GET_VM";
+export const GET_VM_SNAPSHOTS = "GET_VM_SNAPSHOTS";
```

Comment:
You have GET_VM_SNAPSHOTS but UPDATE_DOMAIN_SNAPSHOTS. 
Would be mind choosing VM or DOMAIN for consistency?

Reply:
Thanks. Gonna use DOMAIN_SNAPSHOT

Reply:
Fixed last two comments, but I don't know how to do the first thing (Promises.All) properly in this case,